sdk/core/core-https/src/constants.ts

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export const SDK_VERSION: string = "1.0.0";
```

Comment:
Nit: I know this doesn't really matter until we start using this in clients, but should this be marked with a preview version?

Reply:
Updated for safety purposes